### PR TITLE
feat: show unconfirmed balance

### LINKED ISSUE #1596: The unconfirmed balance should be shown separately

### PR DESCRIPTION
**Pull request by gabaldon**
close #1596

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -371,12 +371,27 @@ function computeTransactionAddress(inputs, outputs, type) {
...
+  const confirmedBalance = response.result.confirmed
+  const localBalance = response.result.local
+  const unconfirmedBalance = response.result.unconfirmed
+  const accumulatedBalance = new BigNumber(unconfirmedBalance.available)
+    .plus(unconfirmedBalance.locked)
+    .toFixed()
+  const totalAccumulatedBalance = new BigNumber(accumulatedBalance)
+    .plus(localBalance)
+    .toFixed()
+  const totalAvailable = new BigNumber(confirmedBalance.available)
```
Comment (Tommytrg) [inline src/api.js:383]:
```suggestion
  const totalBalance = new BigNumber(confirmedBalance.available)
```

File: src/locales/en.json [hunk 2]
```diff
@@ -194,8 +203,15 @@
...
-  "loading": ["Sheikah is loading your wallet", "This will take a few seconds"],
+  "drag_xprv": [
+    "Drag your",
+    "file here or",
+    "click to import"
+  ],
+  "loading": [
+    "Sheikah is loading your wallet",
+    "This will take a few seconds"
+  ],
```
Comment (Tommytrg) [inline src/locales/en.json:214]: Don't format `src/locales` to avoid conflicts with crowding PRs

File: src/locales/en.json [hunk 3]
```diff
@@ -264,11 +280,26 @@
...
+    "Source Scripts"
+  ],
+  "editor_stage_4": [
+    "Set",
+    "Aggregator"
+  ],
+  "editor_stage_5": [
+    "Set",
+    "Tally"
+  ],
```
Comment (Tommytrg) [inline src/locales/en.json:302]: Don't format `src/locales` to avoid conflicts with crowding PRs

File: src/locales/en.json [hunk 4]
```diff
@@ -372,5 +403,6 @@
   "block_notifications_title": "Block notifications",
   "tx_notifications_title": "Transactions notifications",
   "payments_notifications_title": "Payments notifications",
-  "syncronization_notifications_title": "Syncronization notifications"
-}
+  "syncronization_notifications_title": "Syncronization notifications",
+  "unconfirmed": "Unconfirmed"
+}
```
Comment (Tommytrg) [inline src/locales/en.json:408]: Don't format `src/locales` to avoid conflicts with crowding PRs

File: src/locales/es.json [hunk 5]
```diff
@@ -372,5 +372,6 @@
   "block_notifications_title": "Block notifications",
   "tx_notifications_title": "Transactions notifications",
   "payments_notifications_title": "Payments notifications",
-  "syncronization_notifications_title": "Syncronization notifications"
+  "syncronization_notifications_title": "Syncronization notifications",
+  "unconfirmed": "Unconfirmed"
```
Comment (Tommytrg) [inline src/locales/es.json:376]: Don't format `src/locales` to avoid conflicts with crowding PRs

**Comment by clbartoli** [comment]
I think the total balance should include the unconfirmed amount

image

**Review by Tommytrg — APPROVED** [review]

**Pull request merged**